src/components/deprecated-warning/__stories__/deprecated-warning.mdx

```diff
@@ -0,0 +1,14 @@
+import { Controls, Canvas, Meta } from '@storybook/blocks';
+import * as DeprecatedWarningStories from './deprecated-warning.stories';
+
+<Meta of={DeprecatedWarningStories} />
+
+# DeprecatedWarning
+
+## Overview
+
```

Comment:
Maybe let's write here that it's a version of Tip component and give a link to it?